vignettes/performance_analysis.Rmd

```diff
@@ -0,0 +1,323 @@
+
+A successful voting rights case may result in a local jurisdiction's districting map to be thrown out, prompting the need for a new map. Thus, it's necessary assess whether the new map provides sufficient representation for minority groups. It’s not desirable to simply wait for elections to happen and see what the results might be. Instead, we can look at past elections, and observe how candidates would perform if the proposed map was used at the time of the election. This is the basis of a performance analysis.
+
+Ultimately, we need to assess the demographic breakdown across racial groups for each district in the new map. What data source do we use to calculate the percentage of each racial group? We could use total population or citizen voting age population (CVAP), as provided by the Census Bureau. However, these measures may not be accurate, because they assume that voter turnout is equal across racial groups. This is not always true, especially in cases of racial gerrymandering, where turnout for minority groups may be depressed. Instead, we argue that going to the level of the _voter file_ is necessary, as the voter file actually informs _who turned out to vote_.
+
+## The steps in a performance analysis
+
+To conduct a performance analysis, we need to join the voters in a voter file to the new districts and determine the turnout by race, per district. This includes the following steps:
+
+1. **Geocoding:** Geocode latitude and longitude of voters in the file using their address. We do not focus on this step in this vignette.
+2. **Spatial Join:** Using the geocoded coordinates, map each voter to the new districts using a spatial join.
+3. **Race Estimates:** If race is not recorded on the voter file, estimate it using Bayesian Improved Surname Geocoding (BISG) (which itself relies on a spatial join between the voters' coordinates and Census blocks).
+4. **Aggregation:** Aggregate the race estimates per district to obtain voter turnout, by race, in each district.
+
+`eiCompare` provides functions to perform steps 2-4 of the analysis, as well as a function that completes the entire pipeline. Note that Gecoding (step 1) must be performed separately (`eiCompare` provides tools to aid in geocoding: see the Geocoding vignette). In the following section, we walk through each of the steps.
+
+## Case study: East Ramapo School District
+
+The example we'll use to demonstrate the performance analysis is East Ramapo School District (ERSD), located in Rockland County of the New York City suburbs. ERSD is highly segregated, with the majority of Black and Hispanic students attending public schools and white students attending private schools. Furthermore, its voting system for the School Board was at-large, where all voters could vote for all seats on the school board. This system favored the white families whose students largely attended private schools, resulting in redistribution of funds that had adverse impacts on public school students.
```

Comment:
Furthermore, ERSD uses an at-large voting system for School Board elections, where all voters could...